app/app.yaml

```diff
@@ -0,0 +1,39 @@
+components:
+  schemas:
+    App:
+      description: >-
+        A container for emulated protocol and traffic configuration.
```

Comment:
"emulated protocol" description can also apply to "devices" in the OTG tree.  Need to be more precise in what this is and how it is different from devices.